models/classes/dataBinding/class.GenerisInstanceDataBinder.php

```diff
@@ -129,13 +122,13 @@ public function bind($data)
                             );
                         }
                     } elseif (is_string($propertyValue)) {
-                        $instance->editPropertyValues(
-                            $prop,
-                            $propertyValue
-                        );
-                        if (strlen(trim($propertyValue)) == 0) {
-                            //if the property value is an empty space(the default value in a select input field), delete the corresponding triplet (and not all property values)
-                            $instance->removePropertyValues($prop, ['pattern' => '']);
+                        if($this->isEmptyValue($propertyValue)) {
+                            $instance->removePropertyValues($prop);
+                        } else {
+                            $instance->editPropertyValues(
+                                $prop,
+                                $propertyValue
+                            );
```

Comment:
Missing small space. 

```suggestion
                        if ($this->isEmptyValue($propertyValue)) {
                            $instance->removePropertyValues($prop);
                        } else {
                            $instance->editPropertyValues(
                                $prop,
                                $propertyValue
                            );
```

Reply:
Fixed in fdeb3539eb4014036702bc8089116519706a6a43